ray_ci/test_pipeline_ci.py

```diff
@@ -230,5 +233,49 @@ def test_read_lineline():
         assert steps == [{"name": "foo"}]
 
 
+def test_pipeline_ci():
+    env = os.environ.copy()
+    env.update({
+        "REMOTE_CACHE_URL": "https://remote-cache",
+        "ECR_REPO": "ecr_repo",
+        "ECR_BASE_REPO": "ecr_base_repo",
+        "RUNNER_QUEUE_DEFAULT": "default",
+        "RUNNER_QUEUE_SMALL": "small",
+        "RUNNER_QUEUE_MEDIUM": "medium",
+        "RUNNER_QUEUE_LARGE": "large",
+        "RUNNER_QUEUE_GPU_NORM": "gpu",
+        "RUNNER_QUEUE_GPU_LARGE": "gpularge",
+        "BUCKET_PATH": "s3://bucket",
+        "RAY_REPO_DIR": "/ray",
+        "PIPELINE_REPO_DIR": "/pipeline",
+
+        "RAY_CI_PYTHON_DEPENDENCIES_AFFECTED": "1",
+        "BUILDKITE_PULL_REQUEST_REPO": "ray",
+        "BUILDKITE_BRANCH": "master",
+        "BUILDKITE_COMMIT": "abcd1234",
```

Comment:
Is this hard-coded?

Reply:
this is to provide the required env var input for the script to run.